sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/command_component.py

```diff
@@ -151,15 +173,19 @@ def _get_origin_code_value(self) -> Union[str, os.PathLike, None]:
 
     @property
     def instance_count(self) -> int:
-        """Return value of promoted property resources.instance_count.
+        """The number of instances or nodes to be used by the compute target.
 
-        :return: Value of resources.instance_count.
-        :rtype: Optional[int]
+        :rtype: int
         """
         return self.resources.instance_count if self.resources else None
 
     @instance_count.setter
-    def instance_count(self, value: int):
+    def instance_count(self, value: int) -> None:
+        """Sets the number of instances or nodes to be used by the compute target.
+
+        :param value: Tne number of instances of nodes to be used by the compute target. Defaults to 1.
```

Comment:
Nit Typo: Tne
```suggestion
        :param value: The number of instances of nodes to be used by the compute target. Defaults to 1.
```

Reply:
Unsettling that cspell didn't fail on this. Good catch!